.eslintrc.json

```diff
@@ -7,6 +7,7 @@
     "jsx-a11y/label-has-for":0,
     "react/jsx-closing-tag-location":0,
     "import/no-named-as-default":0,
+    "import/prefer-default-export":0,
```

Comment:
i think this rule should be there rather than being ignored by the linter

Reply:
Let me look into it thanks